main/AndroidManifest.xml

```diff
@@ -947,6 +949,7 @@
         <package android:name="googoo.android.btgps" />
         <package android:name="menion.android.locus" />
         <package android:name="menion.android.whereyougo" />
+        <package android:name="net.osmand" />
```

Comment:
net.osmand.plus missing

Reply:
hmpf.

added it :-)

Reply:
Does this support some placeholders or regex patterns? I guess not...

Reply:
not that I know of

and no string placeholders as well, sadly - would be nice, as we have many of those package names already defined in `strings_not_translatable.xml`